### PR TITLE
Minor modifications due to code review

### PR DESCRIPTION
**Pull request by francischeung**
I think I fixed the major issues raised by Ryan.

### CODE REVIEW

**Comment by dragon119** [comment]
It looks good.  Most of the changes seem to be organizing parameters in a way users can see their usage.

File: 3tier-windows/3TierCLIScript.cmd [hunk 1]
```diff
@@ -1,21 +1,8 @@
...
-:: is active/default
-
-SET SUBSCRIPTION=%1
-SET ADMIN_ADDRESS_PREFIX=%2
-SET PASSWORD=%3
-
-:: Set up variables to build out the naming conventions for deploying
-:: the cluster
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+:: Set up variables to for deploying resources to Azure.
```
Comment (MikeWasson) [inline 3tier-windows/3TierCLIScript.cmd]: "to for" > "for"

Comment (MikeWasson) [inline 3tier-windows/3TierCLIScript.cmd]: Can we add a note "Change these variables for your own deployment"?

**Comment by MikeWasson** [comment]
looks good!

**Pull request merged**